docs/manual/meta/passthrough.rst

```diff
@@ -0,0 +1,243 @@
+Meta Passthrough
+================
+
+If your project requires the user to see their physical environment, you'll need to use the headset's passthrough functionality.
+The Godot OpenXR Vendors Plugin provides access to several passthrough modes and filters supported on the Meta Quest, all of which will be broken down in this tutorial.
+
+Export Settings
+-----------------
+
+In order to utilize passthrough, you'll first have to enable the feature in your project's Export Settings. Open up **Export Settings**, navigate to **Meta XR Features**,
+and locate the **Passthrough** setting. Changing this to either **Optional** or **Required** will allow a user's headset to utilize passthrough if permission is granted.
+
+.. image:: img/passthrough/passthrough_export_settings.png
+
+.. _reconstruction-passthrough:
+Reconstruction Passthrough
+--------------------------
+
+.. figure:: img/passthrough/reconstruction_passthrough.jpg
+    :align: center
+
+    Example of unfiltered reconstruction passthrough imagery (Quest Pro)
+
+Reconstruction passthrough will fill the entire background with imagery of the user's surrounding environment, thus "reconstructing" the space outside of the headset.
+To enable this passthrough mode, ensure the following:
+
+* The viewport's ``transparent_bg`` property is set to ``true``.
+* The environment's ``background_mode`` property is set to ``Environment.BG_COLOR``.
+* The environment's ``background_color`` property should have an alpha value of ``0.0``.
+* The `OpenXRInterface <https://docs.godotengine.org/en/stable/classes/class_openxrinterface.html>`_'s ``environment_blend_mode`` property is set to ``XRInterface.XR_ENV_BLEND_MODE_ALPHA_BLEND``.
+
+The ``OpenXRInterface`` parent class `XRInterface <https://docs.godotengine.org/en/stable/classes/class_xrinterface.html>`_ has a method ``get_supported_environment_blend_modes()``
+that will list ``XRInterface.XR_ENV_BLEND_MODE_ALPHA_BLEND`` if it is supported. To enable reconstruction passthrough, you might add the following to the script attached to the root node of your main scene:
+
+.. code-block:: gdscript
+
+    extends Node3D
+
+    @onready var world_environment: WorldEnvironment = $WorldEnvironment
+
+    func _ready():
```

Comment:
Let's use another method instead of `_ready`, since the user may want to enable passthrough at other times besides initialization.
Something like `_enable_passthrough(enable: bool) -> bool`